collectors/InventoryCollector.py

```diff
@@ -11,10 +11,9 @@ class InventoryCollector(BaseCollector):
     def __init__(self):
         super().__init__()
         self.name = self.__class__.__name__
-        self.wait_for_inventory_data()
 
     def describe(self):
-        self.collect()
+        return self.collect()
```

Comment:
Funny this went in the code in the first place. No need to have `describe()` in here if we are calling `collect()`, that's done by prometheus implicitly if describe is missing.

Reply:
Please announce the 4 metrics we are using in `collect` here, to avoid initial calling of `collect()`